src/vs/workbench/contrib/notebook/browser/view/renderers/webviewPreloads.ts

```diff
@@ -716,8 +716,18 @@ async function webviewPreloads(ctx: PreloadContext) {
 		};
 	}
 
-	function showPreloadErrors(outputNode: HTMLElement, ...errors: readonly Error[]) {
-		outputNode.innerText = `Error loading preloads:`;
+	async function renderOutputItem(rendererApi: rendererApi.RendererApi, rendererId: string, item: rendererApi.OutputItem, element: HTMLElement, signal: AbortSignal) {
```

Comment:
Eventually I'm planning to have both markup and output elements go through the same rendering pipeline

For legacy reasons, we currently have two pipelines that do different things